### ISSUE TITLE
Idea for snippets

### ISSUE THREAD

**Issue by osaris** [hoaproject/W3#39]
Snippets are short code to show how to assemble Hoa libraries to build a new lib or a small app. We need something like 20-30 snippets to show randomly on homepage. 

This will help us promote the concept of "Compose and extend libraries to create applications or your own libraries!".

Please post snippets ideas and @Hywan (or other motivated @hoaproject/hoackers) will implement them   (maybe in a separate repository).

**Comment by Hywan** [comment]
/ping @guiled (because you're the author of this idea during Apex'14).

**Comment by camael24** [comment]
Made an cURL library with hoa :dancer:

**Comment by Hywan** [comment]
@camael24 Are you crazy?

**Comment by camael24** [comment]
Not really, but i probably use bad word,i mean use "HTTP" , forge a request, send it over http, and parse the response :+1:
